src/crafting.rs

```diff
@@ -277,6 +278,7 @@ pub fn calculate_crafting_profit(
             listing_profit += profit;
             total_crafting_cost += crafting_cost;
             crafting_count += output_item_count;
+            min_price = buy_price / output_item_count;
```

Comment:
It doesn't seem like `min_price` needs to be stored outside the loop, could it be moved inline i.e. `api::trading_post_price_for_revenue(buy_price / output_item_count)`?

Reply:
`buy_price` and `output_item_count` are not available outside the loop, and `min_price` must be passed to `ProfitableItem` which is created outside the loop. It made more sense to me to only call the math function once on the final value instead of for discarded intermediate values.

Reply:
You're right, my bad